lib/active_merchant/billing/gateways/trust_commerce.rb

```diff
@@ -476,7 +476,7 @@ def message_from(data)
       end
 
       def authorization_from(action, data)
-        authorization = data['transid']
+        action == 'store' ? authorization = data['billingid'] : authorization = data['transid']
```

Comment:
authorization can be defined only once:
`authorization = action == 'store' ? data['billingid'] : data['transid']`

Reply:
maybe is a good time for a refactor ?, something like:

```ruby
def authorization_from(action, data)
  case action
  when 'store'
    data['billingid']
  when *VOIDABLE_ACTIONS
    "#{data['transid']}|#{action}"
  else
    data['transid']
  end
end
```

Reply:
Suggestion taken!